### PR TITLE
CNV-20181: cluster-wide crypto policy RN

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): no CP (RN)

Issue: [CNV-20181](https://issues.redhat.com//browse/CNV-20181)

Link to docs preview: https://54562--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-12-release-notes.html#virt-4-12-new

QE review:
- [x] QE has approved this change.

Additional information: N/A

### CODE REVIEW

File: virt/virt-4-12-release-notes.adoc [hunk 1]
```diff
@@ -88,6 +88,7 @@ The SVVP Certification applies to:
 
 
 //CNV-20181 Comply with cluster-wide crypto policy
+* The {VirtProductName} Operator now reads the cluster-wide xref:../security/tls-security-profiles.adoc#tls-profiles-kubernetes-configuring_tls-security-profiles[TLS security profile]  and propagates it to the {VirtProductName} components, including virtualization, storage, networking, and infrastructure.
```
Comment (dbasunag) [inline virt/virt-4-12-release-notes.adoc:91]: Should this mention from where it is being read?

**Review by dbasunag — CHANGES_REQUESTED** [review]

File: virt/virt-4-12-release-notes.adoc (hunk 1 above)
Comment (ousleyp) [reply virt/virt-4-12-release-notes.adoc:91]: That's a good idea; would that be the `APIServer` custom resource?

**Review by dbasunag — APPROVED** [review]
lgtm

**Pull request merged**